README.yaml

```diff
@@ -18,30 +18,42 @@ badges:
 - name: Slack Community
   image: https://slack.cloudposse.com/badge.svg
   url: https://slack.cloudposse.com
-related:
-- name: terraform-terraform-label
-  description: Terraform Module to define a consistent naming convention by (namespace,
-    environment, stage, name, [attributes])
-  url: https://github.com/cloudposse/terraform-terraform-label
 description: |-
   Terraform module designed to generate consistent names and tags for resources. Use `terraform-null-label` to implement a strict naming convention.
 
-  This module generates names using the following convention by default: `{namespace}-{environment}-{stage}-{name}-{attributes}`.
+  This module generates IDs using the following convention by default: `{namespace}-{tenant}-{environment}-{stage}-{name}-{attributes}`.
   However, it is highly configurable. The delimiter (e.g. `-`) is configurable. Each label item is optional (although you must provide at least one).
-  So if you prefer the term `stage` to `environment`
-  you can exclude environment and the label `id` will look like `{namespace}-{stage}-{name}-{attributes}`.
-  - If attributes are excluded but `namespace`, `stage`, and `environment` are included, `id` will look like `{namespace}-{environment}-{stage}-{name}`.
-  - If you want the attributes in a different order, you can specify that, too, with the `label_order` list.
-  - You can set a maximum length for the name, and the module will create a unique name that fits within that length.
+  So if you prefer the term `stage` to `environment` and do not need `tenant`, you can exclude them
+  and the label `id` will look like `{namespace}-{stage}-{name}-{attributes}`.
+  - If `attributes` and `tenant` are excluded but `namespace`, `stage`, and `environment` are included, `id` will look like `{namespace}-{environment}-{stage}-{name}`.
+    Excluding `attributes` is discouraged, though, because attributes are the main way modules modify the ID to ensure uniqueness.
```

Comment:
```suggestion
    Excluding `attributes` is discouraged, though, because attributes are the main way modules modify the ID to ensure uniqueness when provisioning the same resource types.
```